scalastyle-config.xml

```diff
@@ -250,6 +250,14 @@ This file is divided into 3 sections:
     <customMessage>Omit braces in case clauses.</customMessage>
   </check>
 
+  <!-- SPARK-16877: Avoid Java annotations -->
+  <check customId="OverrideJavaCase" level="error" class="org.scalastyle.scalariform.TokenChecker" enabled="true">
+    <parameters><parameter name="regex">^Override$</parameter></parameters>
```

Comment:
BTW, actually I tried `RegexChecker` as well to grep this case but then I found actually I should come up with a complex regular expression for some exceptional cases such as 

- `@Override` in comments

```scala
  /**
...
   *    @Override
   *    public void close(Throwable errorOrNull) {
   *      // close the connection
   *    }
...
```

- `@Override` in codegen

```scala
...
"  @Override public String toString() { return \"" + toStringValue + "\"; }}"
...
```

So, I had to use `TokenChecker`.